web-registry/src/index.tsx

```diff
@@ -42,6 +42,10 @@ const intercomId = process.env.REACT_APP_INTERCOM_APP_ID || '';
 // you can set this flag in local environments if testing changes to sentry.
 // currently we only want this flag set for the production environment.
 if (process.env.REACT_APP_SENTRY_ENABLED) {
+  const defaultSampleRate = '0.2';
```

Comment:
this reduces the amount of "tracing" or in other words the amount of transactions sent from the app to sentry

Reply:
here's some more info about the `BrowserTracing`:
https://docs.sentry.io/platforms/javascript/performance/instrumentation/automatic-instrumentation/

and the trace sampling rate:
https://docs.sentry.io/platforms/javascript/configuration/sampling/#setting-a-uniform-sample-rate

Reply:
That's good to know.